examples/chip-tool/templates/logging/DataModelLogger-src.zapt

```diff
@@ -122,7 +124,7 @@ CHIP_ERROR DataModelLogger::LogCommand(const chip::app::ConcreteCommandPath & pa
             case {{asUpperCamelCase parent.name}}::Commands::{{asUpperCamelCase name}}::Id:
             {
                 {{asUpperCamelCase parent.name}}::Commands::{{asUpperCamelCase name}}::DecodableType value;
-                ReturnErrorOnFailure(chip::app::DataModel::Decode(*data, value));
+                ReturnErrorOnFailure(chip::app::DataModel::Decode(*data, value, kUndefinedFabricIndex));
```

Comment:
This is less than ideal... but not sure what else we can do (`LogAttribute` similarly will log bad data; it's just not obvious because it doesn't have the new method override like commands do)